### PR TITLE
BIG-222 test if allowing delete ex bucket with deleted schema on backend

### PR DESCRIPTION
**Pull request by romanbracinik**
Jira: BIG-222
KBC: XXX

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)
---
este k BIG-222 nebol som si isty ci ide zmazat ex bucket ktoremu odmazem prava na backend tak som rovno pridal do testov

### CODE REVIEW

File: tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php [hunk 1]
```diff
@@ -224,6 +224,13 @@ public function testRefreshBucketWhenSchemaDoesNotExist(): void
         // test bucket still exists
         $bucket = $this->_client->getBucket($idOfBucket);
         $this->assertNotEmpty($bucket);
+
+        $this->_client->dropBucket($idOfBucket);
+
+        // test bucket is deleted
+        $this->expectException(ClientException::class);
```
Comment (jirkasemmler) [inline tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php:231]: prosim zabal to do try-catche. Ty `expectExceptionXXX` bych rad prestal pouzivat, protoze to pak dela bordel, kdybys pod to neco jeste pridaval

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
chapu, ale na co je to dobry?

File: tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php (hunk 1 above)
Comment (romanbracinik) [reply tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php]: as you wish

**Comment by romanbracinik** [comment]
> chapu, ale na co je to dobry?

no chcel som si overit, ze nespadne ten drop a dokazeme zmazat metadata aj ked ta schema neexistuje. Keby to nieje EX a zmizne tomu schema tak to vyfailuje tak to ma overovat ze skipujeme error ked ta schema neexistuje

**Comment by romanbracinik** [comment]
updatli sa asi nejake pravidla phpstanu https://github.com/keboola/storage-api-php-client/pull/1272/commits/d9a984f065c3444b84b6ab257a019097c9a350d8

**Review by jirkasemmler — APPROVED** [review]
fixni si ten stan a cajk

**Pull request merged**